### PR TITLE
Add bilingual feature

### PR DESCRIPTION
**Pull request by talisman6803**
## Pull request checklist

**Please check if your PR meets the following requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Tested with (`npm run dev`) and is working great
- [ ] Build tested with (`node build/build.js`) and no errors
- [ ] Double checked the two items above

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Documentation
- [ ] Other (describe): 

## Describe your Pull Request

Modify ``BigName.vue`` , ``Title.vue`` , ``Target.vue`` for bilingual features.

## Additional content

Due to some sort of commit problem, I did PR again after solving the problem.

### CODE REVIEW

File: src/components/BigName.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <span id = "depart">
-  <p>{{ this.target }}</p>
+  <span id="depart">
+    <p>{{ this.target }}</p>
```
Comment (Taewan-P) [inline src/components/BigName.vue]: This spacebar is unnecessary

File: src/components/BigName.vue [hunk 2]
```diff
@@ -10,34 +10,50 @@ export default {
   props: ["val"],
   data() {
     return {
-      target: this.getStationName(this.val)
+      target: this.getStationName(this.val),
```
Comment (Taewan-P) [inline src/components/BigName.vue]: The comma at the end of this line is unnecessary

File: src/components/BigName.vue [hunk 3]
```diff
@@ -10,34 +10,50 @@ export default {
...
+            return "Shuttlecoke";
+          case "shuttlecock_i":
+            return "Shuttlecoke";
+          default:
+            return "???";
+        }
       }
-    }
-  }
+    },
```
Comment (Taewan-P) [inline src/components/BigName.vue]: The comma at the end of this line is unnecessary too.

File: src/components/BigName.vue [hunk 4]
```diff
@@ -10,34 +10,50 @@ export default {
...
+  },
```
Comment (Taewan-P) [inline src/components/BigName.vue]: The comma at the end of this line is unnecessary too.

File: src/components/BigName.vue [hunk 5]
```diff
@@ -10,34 +10,50 @@ export default {
...
+          case "yesulin":
+            return "예술인";
+          case "shuttlecock_o":
+            return "셔틀콕";
+          case "shuttlecock_i":
+            return "셔틀콕 건너편";
+          default:
+            return "???";
+        }
+      } else {
```
Comment (Taewan-P) [inline src/components/BigName.vue]: Please use a new line when stating else {} statement.

File: src/components/Target.vue [hunk 6]
```diff
@@ -2,15 +2,22 @@
...
-      <span
-        class="time_num"
-      >{{ this.buslist !== "hello" ? (minutes < 10 ? "0" + minutes + " :" : minutes + " :") : "" }}</span>
-      <span
-        class="time_num"
-      >{{ this.buslist !== "hello" ? (seconds < 10 ? "0" + seconds : seconds) : "" }}</span>
+      <span class="time_num">{{
+        this.buslist !== "hello" ? (hours > 0 ? hours + " :" : "") : ""
+      }}</span>
+      <span class="time_num">{{
```
Comment (Taewan-P) [inline src/components/Target.vue]: Why is this element divided in three lines? Please make these in one line.

File: src/components/Target.vue [hunk 7]
```diff
@@ -2,15 +2,22 @@
...
+        this.buslist !== "hello"
+          ? minutes < 10
+            ? "0" + minutes + " :"
+            : minutes + " :"
+          : ""
+      }}</span>
+      <span class="time_num">{{
+        this.buslist !== "hello" ? (seconds < 10 ? "0" + seconds : seconds) : ""
+      }}</span>
```
Comment (Taewan-P) [inline src/components/Target.vue]: Same here.

File: src/components/Target.vue [hunk 8]
```diff
@@ -26,17 +33,18 @@ export default {
   },
 
   methods: {
-    getCurrentTime: x => {
+    getCurrentTime: (x) => {
       x.now = Math.floor(new Date().getTime() / 1000);
-    }
+    },
```
Comment (Taewan-P) [inline src/components/Target.vue]: Unnecessary comma

File: src/components/Target.vue [hunk 9]
```diff
@@ -79,19 +104,19 @@ export default {
...
       if (this.buslist[this.i].type === "F") {
         return 0;
       } else {
-        if (isNaN(this.buslist[this.i].time)) return 0 //prevent NaN text shown
+        if (isNaN(this.buslist[this.i].time)) return 0; //prevent NaN text shown
         return (this.buslist[this.i].time - this.now) % 60;
       }
-    }
-  }
+    },
```
Comment (Taewan-P) [inline src/components/Target.vue]: Unnecessary comma

File: src/components/Target.vue [hunk 10]
```diff
@@ -79,19 +104,19 @@ export default {
...
+  },
```
Comment (Taewan-P) [inline src/components/Target.vue]: Unnecessary comma

**Review by Taewan-P — CHANGES_REQUESTED** [review]
Changes requested below.

**Comment by Taewan-P** [comment]
Please take care of your codes (Indentation, unnecessary codes, etc...) before submitting a PR.

**Comment by talisman6803** [comment]
Please check the new commit. I have corrected the upper matters that are mentioned. It was an unintended problem caused by the personal indentation setting of the editor. Then please let me know if ``Resolve conversion`` is possible.

**Review by Taewan-P — APPROVED** [review]
LGTM

**Pull request merged**